modules/api-import-export/src/main/java/org/wso2/carbon/apimgt/importexport/APIService.java

```diff
@@ -260,4 +262,89 @@ public Response importAPI(@Multipart("file") InputStream uploadedInputStream, @Q
             }
         }
     }
+
+    /**
+     * This service will update an existing API given by the ID. It will preserve state of the API during the update
+     *
+     * @param apiID       ID for the API
+     * @param httpHeaders HTTP headers for authentication mechanism
+     * @return A JSON response indicating the status of the update
```

Comment:
Shall we modify the java docs by defining all the params here?

Reply:
sure